### PR TITLE
Fix Pylint for ingestion/sink

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/src/metadata/ingestion/sink/metadata_rest.py [hunk 1]
```diff
@@ -518,28 +491,6 @@ def delete_table(self, record: DeleteTable):
                 f"Unexpected error deleting table [{record.table.name}]: {exc}"
             )
 
-    def create_lineage_via_es(self, db_schema_and_table, db_schema, db):
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/sink/metadata_rest.py:521]: only used in a function that we need to deprecate, this was just some noise

File: ingestion/src/metadata/ingestion/sink/metadata_rest.py [hunk 2]
```diff
@@ -230,7 +221,13 @@ def write_dashboard_usage(self, dashboard_usage: DashboardUsage) -> None:
             logger.debug(traceback.format_exc())
             logger.error(f"Failed to write dashboard usage [{dashboard_usage}]: {exc}")
 
-    def write_tables(self, db_schema_and_table: OMetaDatabaseAndTable):
+    def write_tables(  # pylint: disable=too-many-branches
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/sink/metadata_rest.py:224]: This we need to deprecate, opened https://github.com/open-metadata/OpenMetadata/issues/7960

File: ingestion/src/metadata/utils/helpers.py [hunk 3]
```diff
@@ -66,6 +67,7 @@
 
 
 def calculate_execution_time(func):
+    @wraps(func)
```
Comment (pmbrull) [inline ingestion/src/metadata/utils/helpers.py:70]: to preserve metadata

**Comment by pmbrull** [comment]
looks like I killed the sample data ingestion, I'll take a look

**Review by ulixius9 — APPROVED** [review]

**Pull request merged**